common/main/java/com/couchbase/lite/Collection.java

```diff
@@ -161,19 +153,15 @@ public ListenerToken addDocumentChangeListener(
 
     @Override
     @NonNull
-    public List<String> indexes() throws CouchbaseLiteException {
```

Comment:
Ha.... I wasn't clear.  I have no problem at all with single line functions.  If you have to break the signature, across lines, though, I would like the body separate.